packages/create-react-app/index.js

```diff
@@ -264,6 +270,28 @@ function checkAppName(appName) {
   }
 }
 
+function moveReactScriptsToDev(packageName) {
+  var packagePath = path.join(process.cwd(), 'package.json');
+  if (!pathExists.sync(packagePath)) {
```

Comment:
I think this check is unnecessary, because we wrote this file ourself earlier. And if it doesn't exist for some weird reason, it's actually better if the `require` call below fails loudly than to fail silently.